File: code/client/economycraft/lib/classes/player.dart
class Player {
  final int id;
  final String name;
  final String deliveryAddress;
  final String avatarUrl;
  final bool ai;
  double money;
  final DateTime createdAt;

  Player({
    required this.id,
    required this.name,
    required this.deliveryAddress,
    required this.avatarUrl,
    required this.ai,
    required this.money,
    required this.createdAt,
  });
}

File: code/server-processes/src/classes/classes/player.py
from uuid import UUID


class Player:
    
    def __init__(self, id: int, user_id: UUID, minecraft_username: str, money: int, avatar_url: str, delivery_address: str) -> None:
        self.id = id
        self.user_id = user_id
        self.minecraft_username = minecraft_username
        self.money = money
        self.avatar_url = avatar_url
        self.delivery_address = delivery_address
        